[2021-03-10 07:03:58,472] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: data_processing.storing_user 2021-01-26T00:00:00+00:00 [queued]>
[2021-03-10 07:03:58,480] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: data_processing.storing_user 2021-01-26T00:00:00+00:00 [queued]>
[2021-03-10 07:03:58,480] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-10 07:03:58,480] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-10 07:03:58,480] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-10 07:03:58,488] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): storing_user> on 2021-01-26T00:00:00+00:00
[2021-03-10 07:03:58,490] {standard_task_runner.py:51} INFO - Started process 42872 to run task
[2021-03-10 07:03:58,496] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'data_processing', 'storing_user', '2021-01-26T00:00:00+00:00', '--job-id', '154', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_processing.py', '--cfg-path', '/tmp/tmpeyay7nbx']
[2021-03-10 07:03:58,496] {standard_task_runner.py:76} INFO - Job 154: Subtask storing_user
[2021-03-10 07:03:58,626] {logging_mixin.py:103} INFO - Running <TaskInstance: data_processing.storing_user 2021-01-26T00:00:00+00:00 [running]> on host airflowvm
[2021-03-10 07:03:58,760] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_processing
AIRFLOW_CTX_TASK_ID=storing_user
AIRFLOW_CTX_EXECUTION_DATE=2021-01-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-26T00:00:00+00:00
[2021-03-10 07:03:58,761] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-10 07:03:58,761] {bash.py:158} INFO - Running command: echo -e ".separator "," 
. import /tmp/users.csv userstwos" | sqlite3 /home/airflow/airflow/airflow.db
[2021-03-10 07:03:58,766] {bash.py:169} INFO - Output:
[2021-03-10 07:03:58,769] {bash.py:173} INFO - /tmp/users.csv:1: expected 3 columns but found 6 - extras ignored
[2021-03-10 07:03:58,773] {bash.py:177} INFO - Command exited with return code 0
[2021-03-10 07:03:58,786] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=data_processing, task_id=storing_user, execution_date=20210126T000000, start_date=20210310T070358, end_date=20210310T070358
[2021-03-10 07:03:58,798] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-10 07:03:58,835] {local_task_job.py:118} INFO - Task exited with return code 0
